introduction-to-datasets.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
+
+URIs also define a name and namespace for your dataset. Kite uses these values when the underlying system has the same concept (for example, Hive). You can retrieve the names and namespaces for the available datasets using the [`Datasets.list`](http://kitesdk.org/docs/current/apidocs/org/kitesdk/data/Datasets.html#list(java.net.URI)) method.
```

Comment:
This is another place where a reference link would be better.

Also the javadoc for `Datasets.list`:
> List the Dataset URIs in the repository identified by the URI

While you can see the names and namespaces, it would be more valuable to call out that the method gives URIs for valid datasets underneath the repository URI. If we are mentioning a repository URI, then we should add it to the URIs document.